engine/src/dskw32.cpp

```diff
@@ -365,7 +365,7 @@ bool MCS_registry_root_to_hkey(MCStringRef p_root, HKEY& r_hkey, uint32_t& x_acc
 		if (MCStringIsEqualToCString(p_root, Regkeys[i].token, kMCCompareCaseless))
 		{
 			r_hkey = Regkeys[i].key;
-			if (MCmajorosversion >= 0x0501)
+			if (MCmajorosversion >= MCOSVersionMake(5,1,0))
```

Comment:
I think this should be `if (MCmajorosversion >= MCOSVersionMake(5,0,1))`

Reply:
I think its correct - windows only actually encoded major.minor before in the hex values - although @livecodeian will need to confirm that

Reply:
Oh I see - so in this case ignore all my other comments, since they are about the same issue :)

Reply:
The version format used on windows was 0xMMmm (MM = major, mm = minor) with no patch / bugfix version number, so 0x0501 => major = 5, minor = 1